### ISSUE TITLE
Accessibility: jQuery UI accordions use `tablist` and `tabpanel` roles incorrectly.

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1965]
[ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/)[WCAG 2.2 A 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html) tabpanel elements must not appear inside the tablist and are usually placed immediately after the tablist

The `ToggleCompositeField` is controlled with a [jQuery UI accordion](https://jqueryui.com/accordion/). That results in a parent element with `role="tablist"`, inside of which both the toggle button (`role="tab"`) and collapsible content (`role="tabpanel"`) are children.

This may be a good opportunity to start moving away from jQuery UI. We shouldn't remove jQuery UI in case people are relying on it for their customisations, but we can stop using it for this.

## Notes

- `ToggleCompositeField` is called out explicitly but check for other cases where we're using jQuery UI for accordion behaviour
- Tabs are probably not the appropriate roles to use for accordion-like behaviour, so check out some existing accessible accordion examples before deciding how to best implement this.
- The copy of jQuery UI we're using is in the `thirdparty` directory of `silverstripe/admin` because we've already made alterations to it. We _could_ just change the roles or markup through edits to jQuery UI directly. However, this is really just kicking the can down the road, as we should be aiming to stop using this library.
- This is an opportunity to provide a JS implementation of this field that would work on the frontend. Consider that option.

## Related

- https://github.com/silverstripe/.github/issues/470

## Acceptance criteria

- [ ] We stop using jQuery UI for any accordion-like behaviour (if sensible - see notes)
- [ ] Accordion-like markup follow accessibility guidelines
- [ ] The changelog clearly calls out any changes to markup selectors so people can adjust their custom css/js as necessary
- [ ] Check `ToggleCompositeField` in both entwine and react forms

**Comment by emteknetnz** [comment]
I had a relatively quick go at just hacking around jQuery UI, it didn't prove as easy as I'd like.

Similar to [replace chosen-js](https://github.com/silverstripe/silverstripe-admin/issues/2027#issuecomment-3483771015), by replacing jQuery UI for accordion behaviour, we're risking double handling here as we're likely to just replace this jquery compnent with another jquery component, only to get rid of it later with a react refactor

Note that there's a [recently raised issue](https://github.com/silverstripe/silverstripe-framework/issues/11875) to add a react version of ToggleCompositeField i.e this component does not currently exist in react

**Comment by GuySartorelli** [comment]
I agree, this is in the too hard basket for now and we should hold off replacing until we know what's happening with CMS 7 JS.
